cmd/registry/cmd/export-csv_test.go

```diff
@@ -35,14 +35,23 @@ func TestExportCSV(t *testing.T) {
 	}
 
 	const (
+		projectID = "export-csv-test-project"
```

Comment:
Could we choose and document a convention for naming projects in tests? A declared namespace (perhaps using a prefix) would help us (occasionally) run tests against running installations and might also improve predictability and help us reduce project name collisions.

I'd suggest calling this "test-export-csv" or (possibly better) "test-registry-export-csv" since it's testing a subcommand of the registry tool.

Reply:
As long as the project names are specific enough not to collide I don't see any additional benefit to documenting and enforcing a common prefix. The project has to be deleted before/after the test run regardless.

If I'm missing the point, can you elaborate on how predictability would increase with a common prefix?

Reply:
Partly it's an aesthetic preference to add some order and predictability by using a naming convention to help future test authors follow a consistent pattern.

It could also help with automation - we might write helpers that delete all "test-" projects, for example.

If we documented that user projects shouldn't use our test prefix, this would allow people to run tests on running instances, and we might further enforce this by explicitly blocking creation of "test-" projects unless a header was set.

Reply:
Makes sense. Having an established pattern makes it easier to follow.

For automation I think it's safer for us to use explicit test/non-prod databases that can be safely wiped out instead of relying on a prefix to indicate which rows are safe to delete. Having the tests connect to a database named `registry_test` makes it unlikely somebody will be upset that its contents are affected by a test run.

This would add a one-time cost to setting up tests to run against Postgres, but it would make this data safety issue extremely obvious to users. I'll think about this idea a bit more because it might solve some of our existing issues with testing too.